src/step/view/step.jsx

```diff
@@ -168,7 +168,7 @@ export default class Step extends Component {
         const { parentWidth, parentHeight } = this.state;
 
         // type不同对应的direction不同
-        direction = shape === 'arrow' ? 'horizontal' : direction;
+        direction = shape === 'arrow' ? 'hoz' : direction;
```

Comment:
this was causing lots of warnings in the unit tests

Reply:
> this was causing lots of warnings in the unit tests

Did you modify this only to fix the warnings in UT? I see this as a business logic or api change, which could case more serious problem instead of just warnings in ut.

Reply:
The warnings are thrown because of a proptypes issue. `horizontal` is not one of the allowed step directions. This seems to be a bug

Reply:
maybe @zobor konws

Reply:
beautifully, i miss this bug, thanks justing